### DISCUSSION TITLE
How to update a single value of an object?

### DISCUSSION THREAD

**Discussion by maltezc** [bugthesystem/FireSharp#155] [General]
Hello, Long story short, I have an object on the real-time database where I would like to update only a single field of that object. Your example shows that you update all fields.

When I try to update a single field, in my case an int field, anything that is not an int or a bool is deleted from my database. 

Is it possible to update a single field? If so, could you please provide an example?

Here is my [Stack Overflow post](https://stackoverflow.com/questions/65895714/c-sharp-firesharp-how-to-update-a-single-value-of-an-object) explaining more.

ps. Thank you for creating Firesharp. huge fan.

**Comment by ayodeko** [comment]
Incase you later solved this, kindly help out with what you did cos I'm in a similar situation
